quality/postman/sor-1-_table.postman_environment.json

```diff
@@ -0,0 +1,49 @@
+{
+	"id": "191023d9-a402-493b-a690-9bcee390571e",
+	"name": "/sor/1/_table/:table",
+	"values": [
+		{
+			"key": "api-key",
+			"value": "local_admin",
+			"enabled": true
+		},
+		{
+			"key": "api-key-no-rights",
+			"value": "should be passed as param in command line",
+			"enabled": true
+		},
+		{
+			"key": "baseUrl",
+			"value": "localhost:8080",
+			"enabled": true
+		},
+		{
+			"key": "baseUrl_EU",
+			"value": "localhost",
+			"enabled": true
+		},
+		{
+			"key": "baseUrl_US",
+			"value": "localhost",
+			"enabled": true
+		},
+		{
+			"key": "write_consistency_weak",
+			"value": "",
```

Comment:
does it make sense to specify value here?

Reply:
Somehow it's got lost while I was preparing this file. I'll update it.